src/compressionLib/compression.ts

```diff
@@ -22,25 +21,25 @@ async function compression(pathToFolder: string, outputPath: string, callback: (
         try {
             const { stdout, stderr } = await exec(`cd "${ROOT_PATH}" && tar cf - "${pathToFolder}" | "${searchConfig.LIBRARY_CONSTANTS.MAC_LIBRARY_FOLDER}/lz4.exec" > "${outputPath}.tar.lz4"`);
             if (stderr) {
-                log.send(logLevels.INFO, `compression stderr: ${stderr}`);
+                logger.info(`compression: compression stderr: ${stderr}`);
```

Comment:
```suggestion
                logger.info(`compression: compression stderr:`,  stderr);
```

All the concatenated values can be passed as the second arg

Reply:
Done 👍